### PR TITLE
Feat/button

### PR DESCRIPTION
**Pull request by biswanath-tewari**

### CODE REVIEW

File: components/button.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  position: fixed;
+  bottom: 1rem;
+  right: 1rem;
+  z-index: 5;
+  background-color: var(--button-color);
+  color: var(--text-color-dark);
+  border-radius: 50%;
+  width: 3.5rem;
+  height: 3.5rem;
+  display: flex;
```
Comment (chanchal16) [inline components/button.css:67]: If following OOCSS methodology then you can make a separate class for repeating properties.

Comment (iambizan) [reply components/button.css:67]: Thanks for input. DId not make a separate class for flex alignment due to a user's point of view. The end user had to use "btn-floating-action flex-center".

**Comment by Bibin-Antony** [comment]
Code is reviewed, Could move Forward!

**Review by Bibin-Antony — COMMENTED** [review]
Changes reviewed, everything seems fine. Could move forward.

**Pull request merged**